### ISSUE TITLE
[Feature Request] Add an option to use a persistent notification to always check for notifications in the background

### ISSUE THREAD

**Issue by RachaelAva1024** [oxen-io/session-android#1101]
**Code of conduct**

- [x] I have read and agree to adhere to the [Code of Conduct](https://github.com/oxen-io/session-android/blob/master/CODE_OF_CONDUCT.md).

---

I absolutely LOVE Session, and I use it every single day to talk to my friend. But there is one thing about Session that holds it back from being the absolute perfect messenger for us:

The delay in the delivery of Session notifications when outside of the app is absolutely awful! Usually, we will receive each other's notifications about 10 minutes after it's sent. In the worst cases, there will be a delay of 20-30 minutes.

And one time recently, my friend's Session app didn't notify her of any of my messages for an entire day! I legitimately thought something bad had happened to her, and I was worried about her! Turns out she was completely fine, just that the Session app never notified her of any of my messages.

The reason for this is due to us using GrapheneOS, a custom Android ROM without Google Play Services. In the Session app, there is an option to enable Fast Mode, and on Android, this utilizes the Google Play Services, which is something me and my friend refuse to install for privacy and security reasons. Additionally, we use Session's offical F-Droid repository to install the app.

So, I propose a solution to address this issue:

Add an option to integrate a persistent notification in the notification center that allows the app to always check for notifications in the background, similar to how Signal handles notifications.

I understand that many people would be irritated about another persistent notification in the notification center if it was forced, so making that optional is the best middle ground for everyone.

At this point, I'm so done with the delayed notifications that I'm willing to have another persistent notification in the notification center if it means getting notified instantly about any Session messages from my friend and vice versa.

Thank you for developing a private messenger app that truly believes in privacy, and I hope that this option will appear in a release of the app in the near future. Cheers!

**Comment by gamingmsi** [comment]
Agreed. I don't really understand what's the point of using Session if you have Google Services for 'faster notifications'.
